### PR TITLE
tf Classifier

### PR DESCRIPTION
**Pull request by rezhv**
Logic is similar. The purpose is to rewrite training code and try the tensorflow pertained model just to make sure our models and PyTorch code is not flawed. Compare experiment results to PyTorch results. Some of the functionality is not there yet including freezing the layers. Will add in the next version.

### CODE REVIEW

**Comment by AlynaGray** [comment]
I'm not familiar with tensorflow, but it looks good to me. The only thing that I think is incorrect is that you're using binary cross entropy instead of cross entropy. I think that binary cross entropy is meant to be used when each of the labels can be zero or one independently of the the other labels. However, I will accept this pull request and let you experiment with it.
